tests/gitlab/build_and_test

```diff
@@ -145,7 +145,9 @@ then
     make distclean
 
     echo "Installing dependencies."
-    python tests/uberenv/uberenv.py --spec="${spec}" "${upstream_opt}" "${prefix_opt}"
+    command -v python > /dev/null && python_cmd=python || python_cmd=python3
+    $python_cmd tests/uberenv/uberenv.py --spec="${spec}" "${upstream_opt}" \
+        "${prefix_opt}"
```

Comment:
@v-dobrev did you try calling the script directly ?
```
./tests/uberenv/uberenv.py --spec="${spec}" "${upstream_opt}" "${prefix_opt}"
```
Uberenv has a shebang that’s supposed to be working.
```
#!/bin/sh
"exec" "python3" "-u" "-B" "$0" "$@"
```

Reply:
The `uberenv.py` in https://github.com/mfem/mfem-uberenv/blob/toss-4-update/uberenv.py (this is the branch that is currently used) has
```
#!/bin/sh
"exec" "python" "-u" "-B" "$0" "$@"
```
and TOSS-4 has no `python` in the default path. Of course, there are other solutions to this issue -- the above is just one that I picked. If you prefer to use something else, feel free to suggest changes or just push in this branch.

Reply:
OK then go ahead with this. But I suggest updating uberenv to the version 1.0.0 sooner than later.